.spice/datasets/eth_nft_airdrop_transfers.yaml

```diff
@@ -0,0 +1,9 @@
+name: spiceai.datasets.eth_nft_airdrop_transfers
+type: view
+migrations:
+  - name: create_view
+    sql: |
+      SELECT nt.token_address, nt.token_id, t.from_address AS tx_from_address, t."value" AS tx_value, nt.from_address, nt.to_address, nt.transaction_hash, nt.block_number, nt.block_timestamp, nt.block_hash
```

Comment:
typo

Reply:
Which/where is the typo? This is from [`pkg/dremio/datasets/query.yaml`](https://github.com/spicehq/data-platform/blob/d04fc996f5c4f9506afc3816fd2d071d81d763ae/pkg/dremio/datasets/query.yaml#L77)